### DISCUSSION TITLE
Errore Symfony nell'installazione

### DISCUSSION THREAD

**Discussion by dannycastellano** [iisgiua/giuaschool#668] [Richieste di aiuto]
Buongiorno,
sto cercando di installare il registro elettronico tramite l'archivio ZIP.
Sto seguendo alla lettera la vostra guida ma dopo aver decompresso il file nella cartella principale del file manager attraverso FileZilla non riesco a visualizzare il mio sito online ma visualizzo una pagina di errore Symfony "ClassNotFoundError" con scritto: _Attempted to load class "DoctrineFixturesBundle" from namespace "Doctrine\Bundle\FixturesBundle". Did you forget a "use" statement for another namespace?_ con un elenco di posizioni in cui è presente l'errore.

Non riesco a capire dove sbaglio o cosa devo modificare per risolvere il problema. Come hosting uso Altervista.

Se qualcuno riuscisse a darmi una mano su quello che dovrei fare per risolvere il problema ne sarei grado.

Vi ringrazio anticipatamente e aspetto una risposta. 😊

@trinko @fabriziotoppi

**Comment by CobraOfficial01** [comment]
non ti intromettere nelle mie conversazioni

**Comment by ghost** [comment]
@dannycastellano Buongiorno! Avrei bisogno di parlarti, in merito a questa richiesta. Posso parlarti in privato? Mi scrivi alla mia mail: padovesi.roberto@gmail.com? Grazie!
